packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -83,20 +95,40 @@ export const WizardToggle: React.FunctionComponent<WizardToggleProps> = ({
           <CaretDownIcon aria-hidden="true" />
         </span>
       </button>
-      <div className={css(styles.wizardOuterWrap)}>
-        <div className={css(styles.wizardInnerWrap)}>
-          {nav(isNavOpen)}
-          <WizardBody
-            mainComponent={isInPage ? 'div' : 'main'}
-            aria-label={mainAriaLabel}
-            aria-labelledby={mainAriaLabelledBy}
-            hasNoBodyPadding={hasNoBodyPadding}
-          >
-            {activeStep.component}
-          </WizardBody>
+      <WizardDrawerWrapper
+        hasDrawer={hasDrawer && activeStep.drawerPanelContent}
+        wrapper={(children: React.ReactNode) => (
+          <Drawer isInline isExpanded={isDrawerExpanded}>
+            <DrawerContent panelContent={activeStep.drawerPanelContent}>{children}</DrawerContent>
+          </Drawer>
+        )}
+      >
+        <div className={css(styles.wizardOuterWrap)}>
+          <div className={css(styles.wizardInnerWrap)}>
+            <WizardBody
+              mainComponent={isInPage ? 'div' : 'main'}
+              aria-label={mainAriaLabel}
+              aria-labelledby={mainAriaLabelledBy}
+              hasNoBodyPadding={hasNoBodyPadding}
+            >
+              {activeStep.hasOpenDrawerButton && !isDrawerExpanded && (
+                <Button
+                  className="pf-u-float-right pf-u-ml-md"
+                  isInline
+                  variant="link"
+                  onClick={() => onDrawerToggle(!isDrawerExpanded)}
```

Comment:
This is a good point - ideally the drawer should toggle without the user needing to provide the `onDrawerToggle` callback. However since the `isDrawerExpanded` state is managed at the top level by the consumer, it looks like this method needs to stay at the same level in order for the toggle functionality to work as expected.